src/module/account/settings/_component/FormInput/FormInput.tsx

```diff
@@ -0,0 +1,72 @@
+'use client';
+
+import React, { useState } from 'react';
+import {
+  FormFieldButton,
+  FormGroup,
+  FormGroupField,
+  FormGroupLabel,
+} from '@/shared/components/form/FormElements';
+import Error from '@/shared/components/form/Error';
+import styled from 'styled-components';
+import { FieldErrors } from 'react-hook-form';
+import EyeIcon from 'mdi-react/EyeIcon';
+import FormIcon from './_component/FormIcon';
```

Comment:
Whats FormIcom for? Maybe Eyey icon can be part of it?

Reply:
Okkkkk, add Eyey icon into FormIcon Component

Reply:
actually FormIcon has icon wrapper, EyeyIcon has no wrapper

Reply:
okay... sounds complicated... not gonna dig into this